src/cincinnati/mod.rs

```diff
@@ -47,6 +47,10 @@ lazy_static::lazy_static! {
         "zincati_cincinnati_booted_release_is_deadend",
         "Whether currently booted OS release is a dead-end."
     ).unwrap();
+    static ref UPDATE_TARGETS_IGNORED: IntGauge = register_int_gauge!(
+        "zincati_cincinnati_update_targets_ignored_total",
```

Comment:
This is not a total/counter, i.e. it does not accumulate values over time; `ignored_update_targets` would be more suitable. See https://prometheus.io/docs/practices/naming/ for guidance.

Reply:
right, thanks!